### DISCUSSION TITLE
Applying SingleFile multiple times in the same tab/session

### DISCUSSION THREAD

**Discussion by kwstokes** [gildas-lormeau/SingleFile#930] [General]
When I apply the SingleFile save operation (the first time) - either through the button or hotkey, it always works flawlessly.   But frequently the page contents will change with back-end calls to the server.   When I want to save the page again, it usually gets hung in between the "Deferred Images" step - which shows a checkbox, and "Step 1" which is never displayed. 

I read through ALL the discussion to see if someone else has reported this issue, but I didn't see it.   Several discussions suggested turning off "background saving" to fix a wide range of issues.   So I gave that a try also, but it didn't change anything.

I would really like to be able to apply the save operation multiple times.   Does anyone have any thoughts on this?

BTW, I LOVE SingleFile! It has become my GO-TO tool for so many tasks!

**Comment by gildas-lormeau** [comment]
I'm sorry for the late response, do you have an example of a simple page allowing me to reproduce this issue? I've seen this bug in the past with complex pages but it was very hard to determine the cause of the issue because these pages were too complex to debug them.

**Reply by kwstokes** [reply]
I don't know if this counts as a "simple" page, but you can try:
https://www.releases.com/l/all/2022/4/
then scroll (either up or down) and it loads fresh content.

**Reply by gildas-lormeau** [reply]
Thank you, it looks like SingleFile has some issues with frames in Chrome. I cannot reproduce the issue in Firefox though. I'll do some tests, the page looks simple enough. Do yo confirm you're using a Chromium-based browser?

**Reply by kwstokes** [reply]
That's correct.   All my tests were performed using Chrome.

**Reply by gildas-lormeau** [reply]
Thanks for the confirmation :)

**Reply by gildas-lormeau** [reply]
@kwstokes I was able to identify and fix this issue. The fix will be available in the next version of SingleFile.

**Reply by kwstokes** [reply]
That's outstanding!  Thank you so much.
